### PR TITLE
DOCSP-40387 Add Verification section to BI Connector installation pages

### PR DESCRIPTION
**Pull request by kanchana-mongodb**
- [JIRA](https://jira.mongodb.org/browse/DOCSP-40387)
- Stage:
  - [Windows](https://preview-mongodbkanchanamongodb.gatsbyjs.io/bi-connector/DOCSP-40387/tutorial/install-bi-connector-windows/)
  - [Mac](https://preview-mongodbkanchanamongodb.gatsbyjs.io/bi-connector/DOCSP-40387/tutorial/install-bi-connector-macos/)
  - [RHEL](https://preview-mongodbkanchanamongodb.gatsbyjs.io/bi-connector/DOCSP-40387/tutorial/install-bi-connector-rhel/)
  - [Debian](https://preview-mongodbkanchanamongodb.gatsbyjs.io/bi-connector/DOCSP-40387/tutorial/install-bi-connector-debian/)
- [Build](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=6675c521ebce19f314edceea)

### CODE REVIEW

File: source/includes/steps-install-bi-connector-debian.yaml [hunk 1]
```diff
@@ -6,6 +6,72 @@ source:
   ref: download-bi
 ---
 stepnum: 2
+title: Verify the integrity of the downloaded package.
+ref: verify-bi-debian-package
+level: 4
+content: |
+  The MongoDB release team digitally signs all software packages to
```
Comment (jchemburkar) [inline source/includes/steps-install-bi-connector-debian.yaml:13]: Technically the SQL Engines team is the one signing / releasing these packges, we don't go through the release team to do so, so maybe worth updating this line (here and in the subsequent pages)?

**Review by jchemburkar — COMMENTED** [review]
This looks pretty good to me, just had one note! I also forwarded the link to @nbagnard as well to help look over

File: source/includes/steps-install-bi-connector-debian.yaml [hunk 2]
```diff
@@ -6,6 +6,72 @@ source:
...
+  certify that a particular MongoDB package is a valid and unaltered
+  MongoDB release. MongoDB signs each release branch with a different
```
Comment (nbagnard) [inline source/includes/steps-install-bi-connector-debian.yaml:15]:
```suggestion
  MongoDB release. The `bi-connector.asc` key to validate the BI Connector is available on [pgp.mongodb.com](https://pgp.mongodb.com/bi-connector.asc)  
```

File: source/includes/steps-install-bi-connector-debian.yaml [hunk 3]
```diff
@@ -6,6 +6,72 @@ source:
...
+  PGP key in ``.asc`` format.  
+
+  a. Run the following command to downloaded the ``.sig`` file.
```
Comment (nbagnard) [inline source/includes/steps-install-bi-connector-debian.yaml:18]:
```suggestion
  a. Run the following command to download the ``.sig`` file.
```

File: source/includes/steps-install-bi-connector-debian.yaml [hunk 4]
```diff
@@ -6,6 +6,72 @@ source:
...
+       
+     .. code-block:: sh 
+            
+        curl -LO https://info-mongodb-com.s3.amazonaws.com/mongodb-bi/v2/mongodb-bi-linux-{arch}-{platform}-{version}.tgz.sig
```
Comment (nbagnard) [inline source/includes/steps-install-bi-connector-debian.yaml:22]: Maybe we could add an example here. I think it won't be super straightforward for users to figure out the arch and platform.
Here is an example link `https://info-mongodb-com.s3.amazonaws.com/mongodb-bi/v2/mongodb-bi-linux-x86_64-suse12-v2.14.14.tgz.sig`

File: source/includes/steps-install-bi-connector-macos.yaml [hunk 5]
```diff
@@ -6,6 +6,72 @@ source:
   ref: download-bi
 ---
 stepnum: 2
+title: Verify the integrity of the downloaded package.
+ref: verify-bi-mac-package
+level: 4
+content: |
+  The MongoDB release team digitally signs all software packages to
+  certify that a particular MongoDB package is a valid and unaltered
+  MongoDB release. MongoDB signs each release branch with a different
```
Comment (nbagnard) [inline source/includes/steps-install-bi-connector-macos.yaml:15]:
```suggestion
  MongoDB release. The `bi-connector.asc` key to validate the BI Connector is available on [pgp.mongodb.com](https://pgp.mongodb.com/bi-connector.asc)  
```

File: source/includes/steps-install-bi-connector-macos.yaml [hunk 6]
```diff
@@ -6,6 +6,72 @@ source:
...
+  PGP key in ``.asc`` format.  
+
+  a. Run the following command to downloaded the ``.sig`` file. 
+       
+     .. code-block:: sh 
+            
+        curl -LO https://info-mongodb-com.s3.amazonaws.com/mongodb-bi/v2/mongodb-bi-osx-{x86_64|arm64}-{version}.zip.sig
```
Comment (nbagnard) [inline source/includes/steps-install-bi-connector-macos.yaml:22]: Here is are example links : 
- https://info-mongodb-com.s3.amazonaws.com/mongodb-bi/v2/mongodb-bi-osx-arm64-macos11-v2.14.14.zip.sig
- https://info-mongodb-com.s3.amazonaws.com/mongodb-bi/v2/mongodb-bi-osx-x86_64-v2.14.14.zip.sig
```suggestion
        curl -LO https://info-mongodb-com.s3.amazonaws.com/mongodb-bi/v2/mongodb-bi-osx-{x86_64|arm64-macos{11|13}}-{version}.zip.sig
```

File: source/includes/steps-install-bi-connector-macos.yaml [hunk 7]
```diff
@@ -19,15 +85,15 @@ content: |
      
      .. code-block:: sh
 
-        unzip {SOURCE-PATH}/mongodb-bi-osx-x86_64-{version}.zip -d {DESTINATION-PATH}
+        unzip {SOURCE-PATH}/mongodb-bi-osx-{x86_64|arm64}-{version}.zip -d {DESTINATION-PATH}
```
Comment (nbagnard) [inline source/includes/steps-install-bi-connector-macos.yaml:88]:
```suggestion
        unzip {SOURCE-PATH}/mongodb-bi-osx-{x86_64|arm64-macos{11|13}}-{version}.zip -d {DESTINATION-PATH}
```

File: source/includes/steps-install-bi-connector-macos.yaml [hunk 8]
```diff
@@ -19,15 +85,15 @@ content: |
...
 
-  b. Change to the ``mongodb-bi-osx-x86_64-{version}`` directory.
+  b. Change to the ``mongodb-bi-osx-{x86_64|arm64}-{version}`` directory.
```
Comment (nbagnard) [inline source/includes/steps-install-bi-connector-macos.yaml:90]:
```suggestion
  b. Change to the ``mongodb-bi-osx-{x86_64|arm64-macos{11|13}}-{version}`` directory.
```

File: source/includes/steps-install-bi-connector-macos.yaml [hunk 9]
```diff
@@ -19,15 +85,15 @@ content: |
...
      Replace {DESTINATION-PATH} with the path to the directory
      where you extracted the archive in the previous step.
 
      .. code-block:: sh
 
-        cd {DESTINATION-PATH}/mongodb-bi-osx-x86_64-{version}/
+        cd {DESTINATION-PATH}/mongodb-bi-osx-{x86_64|arm64}-{version}/
```
Comment (nbagnard) [inline source/includes/steps-install-bi-connector-macos.yaml:96]:
```suggestion
        cd {DESTINATION-PATH}/mongodb-bi-osx-{x86_64|arm64-macos{11|13}}-{version}/
```

File: source/includes/steps-install-bi-connector-rhel.yaml [hunk 10]
```diff
@@ -6,6 +6,72 @@ source:
   ref: download-bi
 ---
 stepnum: 2
+title: Verify the integrity of the downloaded package.
+ref: verify-bi-rhel-package
+level: 4
+content: |
+  The MongoDB release team digitally signs all software packages to
+  certify that a particular MongoDB package is a valid and unaltered
+  MongoDB release. MongoDB signs each release branch with a different
```
Comment (nbagnard) [inline source/includes/steps-install-bi-connector-rhel.yaml:15]:
```suggestion
  MongoDB release. The `bi-connector.asc` key to validate the BI Connector is available on [pgp.mongodb.com](https://pgp.mongodb.com/bi-connector.asc)  
```

File: source/includes/steps-install-bi-connector-rhel.yaml [hunk 11]
```diff
@@ -6,6 +6,72 @@ source:
...
+  PGP key in ``.asc`` format.  
+
+  a. Run the following command to downloaded the ``.sig`` file.
+       
+     .. code-block:: sh 
+            
+        curl -LO https://info-mongodb-com.s3.amazonaws.com/mongodb-bi/v2/mongodb-bi-linux-{arch}-{platform}-{version}.tgz.sig
```
Comment (nbagnard) [inline source/includes/steps-install-bi-connector-rhel.yaml:22]: Maybe we could add an example here. I think it won't be super straightforward for users to figure out the arch and platform.
Here is an example link `https://info-mongodb-com.s3.amazonaws.com/mongodb-bi/v2/mongodb-bi-linux-x86_64-rhel90-v2.14.14.tgz.sig`

**Review by nbagnard — CHANGES_REQUESTED** [review]

File: source/includes/steps-install-bi-connector-macos.yaml [hunk 12]
```diff
@@ -6,6 +6,72 @@ source:
...
+
+           gpg: key 1CCF1A1263CDD699: public key "MongoDB BI Connector Release Signing Key <packaging@mongodb.com>" imported
+           gpg: Total number processed: 1
+           gpg:               imported: 1
+
+  #. Run the following command to verify the MongoDB installation file.
+
+     .. code-block:: shell
+        
+        gpg --verify mongodb-bi-osx-{x86_64|arm64}-{version}.zip.sig mongodb-bi-osx-{x86_64|arm64}-{version}.zip
```
Comment (nbagnard) [inline source/includes/steps-install-bi-connector-macos.yaml:46]: We missed that one.
```suggestion
        gpg --verify mongodb-bi-osx-{x86_64|arm64-macos{11|13}}-{version}.zip.sig mongodb-bi-osx-{x86_64|arm64-macos{11|13}}-{version}.zip
```

File: source/includes/steps-install-bi-connector-rhel.yaml [hunk 13]
```diff
@@ -6,6 +6,78 @@ source:
   ref: download-bi
 ---
 stepnum: 2
+title: Verify the integrity of the downloaded package.
+ref: verify-bi-rhel-package
+level: 4
+content: |
+  The MongoDB release team digitally signs all software packages to
+  certify that a particular MongoDB package is a valid and unaltered
+  MongoDB release. The `bi-connector.asc` key to validate the BI Connector is available on [pgp.mongodb.com](https://pgp.mongodb.com/bi-connector.asc)
```
Comment (nbagnard) [inline source/includes/steps-install-bi-connector-rhel.yaml:15]: The link look odd on the staging page.
image

File: source/includes/steps-install-bi-connector-debian.yaml [hunk 14]
```diff
@@ -6,6 +6,78 @@ source:
   ref: download-bi
 ---
 stepnum: 2
+title: Verify the integrity of the downloaded package.
+ref: verify-bi-debian-package
+level: 4
+content: |
+  The MongoDB release team digitally signs all software packages to
+  certify that a particular MongoDB package is a valid and unaltered
+  MongoDB release. The `bi-connector.asc` key to validate the BI Connector is available on [pgp.mongodb.com](https://pgp.mongodb.com/bi-connector.asc)
```
Comment (nbagnard) [inline source/includes/steps-install-bi-connector-debian.yaml:15]: The link look odd on the staging page.
image

**Review by nbagnard — CHANGES_REQUESTED** [review]
I just notice a few little things off, but otherwise it's looking good.

**Review by kyuan-mongodb — APPROVED** [review]
LGTM!

**Review by nbagnard — APPROVED** [review]
LGTM!

**Review by jchemburkar — APPROVED** [review]

**Pull request merged**